webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/apidata/JsonResponseObjects.scala

```diff
@@ -397,174 +397,84 @@ object JsonResponseObjects {
         .withFieldComputed(
           _.videos,
           levelField(_)("videos")(
-            inventory.flatMap(_.machine.map(_.videos.transformInto[Chunk[Video]])).filter(_.nonEmpty)
+            inventory.flatMap(_.machine.map(_.videos.transformInto[Chunk[domain.Video]])).filter(_.nonEmpty)
           )
         )
         .withFieldComputed(
           _.virtualMachines,
           levelField(_)("virtualMachines")(
-            inventory.map(_.node.vms.transformInto[Chunk[VirtualMachine]]).filter(_.nonEmpty)
+            inventory.map(_.node.vms.transformInto[Chunk[domain.VirtualMachine]]).filter(_.nonEmpty)
           )
         )
         .buildTransformer
     }
 
-    @jsonDiscriminator("type") sealed abstract class OsDetails(
-        val name:          String,
-        val version:       Version,
-        val fullName:      String,
-        val servicePack:   Option[String],
-        val kernelVersion: Version
-    )
-    object OsDetails {
```

Comment:
:heavy_check_mark: This has already been factored to a `JsonOsDetails` for the codec definition in `NodeFacts`